addOns/pscanrulesAlpha/src/main/javahelp/org/zaproxy/zap/extension/pscanrulesAlpha/resources/help/contents/pscanalpha.html

```diff
@@ -80,5 +80,10 @@ <H2>Fetch Metadata Request Headers Scan Rule</H2>
  Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/FetchMetadataRequestHeadersScanRule.java">FetchMetadataRequestHeadersScanRule.java</a><br>
  Alert ID: <a href="https://www.zaproxy.org/docs/alerts/90005/">90005</a>.
 
+<H2>Same Origin Method Execution</H2>
+This scanning rule examines JSONP endpoints to identify user-controllable function names.
+<p>
+Latest code: <a href="https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/SameOriginMethodExecutionScanRule.java">SameOriginMethodExecutionScanRule.java</a>
+
```

Comment:
You added the help link method in Java but I think something was to be added here as well. An anchor and link to the alert ID?